src/pages/publication/contributors_tab/AddContributor.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { FC, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { Button } from '@material-ui/core';
+
+import Modal from '../../../components/Modal';
+import { Authority } from '../../../types/authority.types';
+import AddContributorModal from './components/AddContributorModal';
+
+const AddContributor: FC = () => {
+  const { t } = useTranslation('publication');
+
+  const [addedAuthors, setAddedAuthors] = useState<Authority[]>([]);
+  const [open, setOpen] = useState(false);
+
+  const handleOpen = () => {
+    setOpen(true);
+  };
+
+  const handleClose = () => {
+    setOpen(false);
+  };
+
+  const addAuthor = (author: Authority) => {
+    setAddedAuthors([...addedAuthors, author]);
+    handleClose();
+  };
+
+  return (
+    <>
+      {addedAuthors && addedAuthors.map(author => <h4 key={author.systemControlNumber}>{author.name}</h4>)}
+      <Button onClick={handleOpen} variant="contained" color="primary" data-testid="add-contributor">
+        {t('contributors.add_author')}
+      </Button>
+      {open && (
```

Comment:
trenger vel ikke `open` både her og på linje 41?

Reply:
jo, fordi når vi kaller addAuthor, så skal modalen lukkes

Reply:
er ikke det formålet til `openModal`-propen til `Modal`? Jeg mener at disse to er like:
```
{open && (
        <Modal openModal={true}>
                  {...}
        </Modal>
)}
```
```
<Modal openModal={open}>
       {...}
</Modal>
```